packages/contracts-bedrock/contracts/dispute/FaultDisputeGame.sol

```diff
@@ -219,17 +301,23 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone {
      * @inheritdoc IDisputeGame
      */
     function resolve() external returns (GameStatus status_) {
+        // TODO: Do not allow resolution before clocks run out.
+
         if (status != GameStatus.IN_PROGRESS) {
+            // If the game is not in progress, it cannot be resolved.
             revert GameNotInProgress();
+        } else if (!claimData[0].countered) {
+            // If the root claim has never been countered, it implicitly wins.
+            status_ = GameStatus.DEFENDER_WINS;
+            status = status_;
+            emit Resolved(status_);
+            return status_;
```

Comment:
this is kinda weird, named return arg and also `return`

Reply:
Do you prefer the updated version?

Reply:
I think this is reasonable. If we find it to be illegible later on we can change, but for now it works and doesn't waste gas iiuc